commands/unpack.go

```diff
@@ -118,7 +119,13 @@ func extractLayer(dest string, layer v1.Layer, bar *mpb.Bar, chown bool) error {
 			if err := os.Mkdir(dir, fi.Mode()&os.ModePerm); err != nil {
 				return err
 			}
-
+			// Fix issue introduced by https://github.com/concourse/registry-image-resource/pull/284
```

Comment:
We dont really need to add comment here for the fix. Instead we should make it as detail as possible in this PR thread(commit message and PR description).

For example you could add some details about how the linked PR introduced the bug e.g. removing a dir and adding it back without owner info (feel free to use whatever you think that'd be more clear).

Thx!

Reply:
Sure, I will remove this comment.